### DISCUSSION TITLE
Extension host keeps crashing if a button in the UI is clicked leaving the webview blank

### DISCUSSION THREAD

**Discussion by paulober** [microsoft/vscode-discussions#1605] [Extension Development QnA] [answered]
Hi, I have a bug that one button in the webview will cause the extension host to crash with the only log this event produces being an entry in "Main":
```console
2024-09-05 19:44:05.219 [error] CodeWindow: failed to load (reason: ERR_BLOCKED_BY_CLIENT, code: -20)
2024-09-05 19:45:15.764 [info] Extension host with pid 34964 exited with code: 0, signal: unknown.
2024-09-05 19:45:15.764 [error] [UtilityProcess id: 68, type: extensionHost, pid: 34964]: crashed with code 1 and reason 'killed'
2024-09-05 19:45:33.178 [error] Blocked vscode-webview request vscode-webview://07ah2qk3gn0a6knrq6q3pnd6p9d310f8gqc9g48lfsosvfbe2dc2/index.html?some-radio=0&different-radio=0&new-radio=0
```
It first happens if I leave a UL positioned absolute open and then click the button but now i got the debugger process running somehow in a state that if i restart and click the button without doing anything it makes the extension host crashing.
This can be resolved by restarting my host vscode instance. But the first occurrence is still a source for this crash.

Does someone has a hint what the issue could be or where to find out more?

**Comment by tamuratak** [comment, marked as answer]
It would be a bug of VS Code. It is appropriate to report it to https://github.com/microsoft/vscode with reproducible steps.

**Comment by paulober** [comment]
Anyone having the same issue I want to note that the issue (in my case) was caused by having a `<form>` tag in the webview HTML. This will instruct the webview to crash without error if you do some things like not stopping some click events before they reach the form or issues with input element if they contain text of a user and you update them with js and stuff like that.

**Reply by alefragnani** [reply]
Hi @paulober ,

Glad to know you found the cause, and thanks for sharing. Btw, if you have opened an issue in VS Code repo, I would ask you to share the same info there, so the team would be notified too.

Thank you

**Reply by paulober** [reply]
They were the one who suggested me, that this could be the issue. And it turned out to be true.
